pkg/engine/deployment.go

```diff
@@ -173,6 +175,11 @@ func newDeployment(ctx *Context, info *deploymentContext, opts deploymentOptions
 		}
 		for i := range opts.imports {
 			imp := &opts.imports[i]
+			_, err := tokens.ParseTypeToken(imp.Type.String())
+			if err != nil {
+				return nil, errors.New(fmt.Sprintf("import type %q is not a valid resource type - "+
+					"refer to the import section of the provider resource documentation.", imp.Type.String()))
```

Comment:
Should we note explicitly that type tokens must be of the form `<package>:<module>:<type>`?

Reply:
So I thought about that but then I sort of thought how a user would think about turning:

```
aws:ec2/instance/Instance
```

into

```
aws:ec2/instance:Instance
```

Do you think `<package>:<module>:<type>` would help them see the differences?